### DISCUSSION TITLE
LVM-Thin Storage

### DISCUSSION THREAD

**Discussion by daschmidt1994** [tteck/Proxmox#2232] [Q&A] [answered]
When I try install a script I can only chose from Typ dir or NFS but can't use a LVM-Thin.
Is this a bug or my fault?

**Comment by tteck** [comment]
Do you mean local-lvm?

**Reply by daschmidt1994** [reply]
I'm new to proxmox and try if this is something for me. Now I saw your scripts and they look very intresting.
Have a proxmox insatllation on a nvme. In the pc is also a SSD. I will use the SSD to store data from the container.
![proxmox](https://github.com/tteck/Proxmox/assets/53835578/bf4d0e57-77fa-4f7c-9287-22a1a12f8ee2)

**Reply by tteck** [reply, marked as answer]
That's asking where to store the **container template**, select local. Next It'll ask where to store the **container**
